app/js/lib/common/errors.js

```diff
@@ -0,0 +1,8 @@
+export function HttpError(statusCode, message) {
```

Comment:
How do you see the future of this file? What other types of error are going to be added later?
At this stage, it looks a bit overkilling to me.